docs/source/10_deployment/01_deployment_guide.md

```diff
@@ -15,9 +15,10 @@ We also provide information to help you deploy to the following:
 * to [Kubeflow Workflows](06_kubeflow.md)
 * to [AWS Batch](07_aws_batch.md)
 * to [Databricks](08_databricks.md)
+* to [Dask](12_dask.md)
 
 <!--- There has to be some non-link text in the bullets above, if it's just links, there's a Sphinx bug that fails the build process-->
 
 In addition, we also provide instructions on [how to integrate a Kedro project with Amazon SageMaker](09_aws_sagemaker.md).
 
-![](../meta/images/deployments.png)
+![](../meta/images/deployments.png)  <!-- TODO(deepyaman): Add Dask to deployment flowchart. -->
```

Comment:
Not sure how to recreate the flowchart; any help would be much appreciated!

Reply:
On an unrelated note, why do all these images not have alt text? I'm not expert on this, but I feel like that's bad for accessibility.

Reply:
@limdauto has the draw.io diagram been committed to the repo?

Reply:
I think for the alt text, we just haven't been consistent/forgot to add it.

Reply:
@datajoely sorry I missed this ping. This flowchart was made by @stichbury iirc, not me :(